lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -91,6 +92,9 @@ class RenderBlockingResources extends Audit {
     const metricComputationData = {trace, devtoolsLog, simulator, settings: metricSettings};
     // @ts-ignore - TODO(bckenny): allow optional `throttling` settings
     const fcpSimulation = await artifacts.requestFirstContentfulPaint(metricComputationData);
+    if (!traceOfTab.timestamps.firstContentfulPaint) {
```

Comment:
@patrickhulce this one I was least sure about because it could have been an extra guard vs needed for correct behavior.

This arguably "works" if `fcpTsInMs` ends up as `NaN` as no resources will be ignored. I could change it to make that explicit instead of throwing instead.